### PR TITLE
Apply feature flag for new link styles

### PR DESCRIPTION
**Pull request by owenatgov**
## What
Applies the feature flag for the new link styles from the latest release of GOV.UK Frontend and fixes a couple of links to ensure they are utilising these new link styles.

## Why
So that this feature can start adding value to users.

[Card](https://trello.com/c/DUGRe64q/752-update-frontend-app-to-use-new-link-styles)

## Visual changes
The following table additionally includes the pages that I tested against for review purposes.

| Page | Before | After |
| --- | --- | --- |
| https://www.gov.uk/ | Screenshot 2021-06-07 at 16 19 16 | Screenshot 2021-06-07 at 16 19 28 |
| https://www.gov.uk/bank-holidays | Screenshot 2021-06-07 at 16 20 14 | Screenshot 2021-06-07 at 16 20 24 |
| https://www.gov.uk/when-do-the-clocks-change | Screenshot 2021-06-07 at 16 20 44 | Screenshot 2021-06-07 at 16 20 58 |
| https://www.gov.uk/done/find-driving-schools-and-lessons | Screenshot 2021-06-07 at 16 21 29 | Screenshot 2021-06-07 at 16 21 46 |
| https://www.gov.uk/find-local-council | Screenshot 2021-06-07 at 16 22 08 | Screenshot 2021-06-07 at 16 22 17
| https://www.gov.uk/help/cookies | Screenshot 2021-06-07 at 16 22 41 Screenshot 2021-06-07 at 16 23 27 | Screenshot 2021-06-07 at 16 24 03 Screenshot 2021-06-07 at 16 24 12 |
| https://www.gov.uk/busking-licence | Screenshot 2021-06-07 at 16 24 55 | Screenshot 2021-06-07 at 16 25 04 |
| https://www.gov.uk/pay-council-tax | Screenshot 2021-06-07 at 16 25 32 | Screenshot 2021-06-07 at 16 25 46 |
| https://www.gov.uk/register-offices | Screenshot 2021-06-07 at 16 26 24 | Screenshot 2021-06-07 at 16 26 38 |
| https://www.gov.uk/roadmap | Screenshot 2021-06-07 at 16 27 21 | Screenshot 2021-06-07 at 16 27 28 |
| https://www.gov.uk/contact-ukvi-inside-outside-uk/y/outside-the-uk/something-else | Screenshot 2021-06-07 at 16 28 15 | Screenshot 2021-06-07 at 16 28 23 |
| https://www.gov.uk/renew-driving-licence-at-70/track | Screenshot 2021-06-07 at 16 28 59 | Screenshot 2021-06-07 at 16 29 09 |
| https://www.gov.uk/foreign-travel-advice | Screenshot 2021-06-07 at 16 29 27 Screenshot 2021-06-07 at 16 29 38 | Screenshot 2021-06-07 at 16 29 45 Screenshot 2021-06-07 at 16 29 56 |

### CODE REVIEW

File: app/views/homepage/index.html.erb [hunk 1]
```diff
@@ -27,13 +27,15 @@
...
           <div class="home-top__links">
             <h2 class="home-top__links-title"><%= t('homepage.index.popular') %></h2>
-            <ul class="home-top__links-list">
-              <li class="home-top__links-item"><a href="/guidance/covid-19-coronavirus-restrictions-what-you-can-and-cannot-do" class="home-top__links-link"><%= t('homepage.index.covid') %></a></li>
-              <li class="home-top__links-item"><a href="/brexit" class="home-top__links-link"><%= t('homepage.index.brexit') %></a></li>
-              <li class="home-top__links-item"><a href="/personal-tax-account" class="home-top__links-link"><%= t('homepage.index.tax_account') %></a></li>
-              <li class="home-top__links-item"><a href="/jobsearch" class="home-top__links-link"><%= t('homepage.index.job_search') %></a></li>
-              <li class="home-top__links-item"><a href="/sign-in-universal-credit" class="home-top__links-link"><%= t('homepage.index.universal_credit') %></a></li>
-            </ul>
+            <%= render "govuk_publishing_components/components/list", {
```
Comment (chris-gds) [inline app/views/homepage/index.html.erb:30]: Nice! 

(Was looking at this recently and also eyeing up the heading above to swap out but the non-bold Design change + being the home page made me hold off...)

**Comment by chris-gds** [comment]
@owenatgov Checking a _good_ selection of `frontend` pages the vast bulk all look ace, loving this new link style. 🙌🏼

Noticed the logo underline doesn't updated but checking the `page_layout` it's there, so once this gem swap is done this will pull through.

Also, I think IE11 needs a revisit  - some links don't change.  I think for ease it might make sense to keep the gem swap for the "popular links" but mimc the current Design for the homepage in terms of the visuals.

I reviewed the Action Link one as well - it seems like this is pre-existing so I [raised an issue:](https://github.com/alphagov/govuk_publishing_components/issues/2125) I have a recollection Action Links were being worked on recently was this part of this?
 
https://user-images.githubusercontent.com/71266765/121356720-06365c00-c929-11eb-9beb-bae6f345fd07.mov

**Comment by owenatgov** [comment]
@chris-gds [`text-decoration-thickness` isn't supported by IE11](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-thickness). This feature is an enhancement for newer browsers. You make a good point on these needing some sort of hover colour for additional affordance though. Will add something in.

**Review by chris-gds — APPROVED** [review]
🎉

**Comment by chris-gds** [comment]
IE11's all good 
Jun-15-2021 16-37-41

**Pull request merged**